pkg/signaling/mcast/backend.go

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: 2025 Adam Rizkalla <ajarizzo@gmail.com>
+// SPDX-License-Identifier: Apache-2.0
+
+package mcast
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"syscall"
+
+	"cunicu.li/cunicu/pkg/crypto"
+	"cunicu.li/cunicu/pkg/log"
+	signalingproto "cunicu.li/cunicu/pkg/proto/signaling"
+	"cunicu.li/cunicu/pkg/signaling"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/proto"
+)
+
+func init() { //nolint:gochecknoinits
+	signaling.Backends["multicast"] = &signaling.BackendPlugin{
+		New:         NewBackend,
+		Description: "Multicast",
+	}
+}
+
+type Backend struct {
+	signaling.SubscriptionsRegistry
+
+	conn       *net.UDPConn
+	mcast_addr *net.UDPAddr
+	config     BackendConfig
+
+	logger *log.Logger
+}
+
+func NewBackend(cfg *signaling.BackendConfig, logger *log.Logger) (signaling.Backend, error) {
+	b := &Backend{
+		SubscriptionsRegistry: signaling.NewSubscriptionsRegistry(),
+		logger:                logger,
+	}
+
+	var err error
+
+	if err = b.config.Parse(cfg); err != nil {
+		return nil, fmt.Errorf("failed to parse backend configuration: %w", err)
+	}
+
+	// Parse multicast group address
```

Comment:
Please end comments with a period.

See: https://go.dev/wiki/CodeReviewComments#comment-sentences

```suggestion
	// Parse multicast group address.
```